cetmix_tower_aws/readme/CONFIGURE.md

```
@@ -0,0 +1,40 @@
+   > Note: These secrets will be accessible as `#!cxtower.secret.aws_access_key!#` and `#!cxtower.secret.aws_secret_access_key!#` in your commands.
+
+3. **Configure AWS Region**
+
+   Create a variable to define your AWS region:
+   
+   - Navigate to `Cetmix Tower > Settings > Variables`
+   - Create a new Variable with:
+     - Name: `AWS Region Name`
+     - Reference: `aws_region_name`
+     - Type: `String`
+   - Set your AWS region (e.g., `us-east-1`, `eu-west-1`) as the value
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)